test/controllers/static_page_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require "test_helper"
```

Comment:
We don't use test folder in project, so add this folder to gitignore and run `git rm --cached -r test`